src/screens/OtherProfile/index.js

```diff
@@ -87,20 +89,23 @@ const OtherProfile = () => {
   const interactionManagerRef = React.useRef(null);
   const [otherProfileFeeds, dispatchOtherProfile] = React.useContext(Context).otherProfileFeed;
   const [profile] = React.useContext(Context).profile;
-  const [, dispatch] = React.useContext(Context).feeds;
-  const [isLastPage, setIsLastPage] = React.useState(false);
+  const [mainFeeds, dispatch] = React.useContext(Context).feeds;
   const [loading, setLoading] = React.useState(false);
   const [initLoading, setInitLoading] = React.useState(true);
   const [isAnonimity, setIsAnonimity] = React.useState(false);
   const {params} = route;
   const {feeds} = otherProfileFeeds;
+  const {timer, viewPostTimeIndex} = mainFeeds;
   const [loadingGenerateAnon, setLoadingGenerateAnon] = React.useState(false);
   const [anonProfile, setAnonProfile] = React.useState();
   const {mappingColorFeed} = useCoreFeed();
   const isSignedMessageEnabled = dataMain.isSignedMessageEnabled ?? true;
   const isAnonimityEnabled = dataMain.isAnonMessageEnabled && isSignedMessageEnabled;
   const {createSignChat} = useCreateChat();
 
+  const {onWillSendViewPostTime} = useViewPostTimeHook(dispatch, timer, viewPostTimeIndex);
+  const {fetchNextFeeds} = useFeedPreloadHook(feeds?.length, () => getOtherFeeds(postOffset));
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `mainFeeds` and `dispatch` are destructured from the context but `dispatch` is not used anywhere in this hunk. If it's not used elsewhere in the code, consider removing it to avoid unnecessary variable declarations.
```diff
- const [mainFeeds, dispatch] = React.useContext(Context).feeds;
+ const [mainFeeds] = React.useContext(Context).feeds;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
closed